File: benchmark/dict_bench.lua
local dict = {}
print("Filling dict with 250,000 keys...")
for i = 0, 249999 do
    dict[tostring(i)] = i
end

local sum = 0
print("Accessing 250,000 keys...")
for i = 0, 249999 do
    sum = sum + dict[tostring(i)]
end
print("Sum: " .. sum)

File: benchmark/dict_bench.py
dict = {}
print("Filling dict with 250,000 keys...")
for i in range(250000):
    dict[str(i)] = i

sum_val = 0
print("Accessing 250,000 keys...")
for i in range(250000):
    sum_val += dict[str(i)]
print("Sum:", sum_val)
